CMake/External_libjpeg-turbo.cmake

```diff
@@ -1,5 +1,17 @@
 # libjpeg-turbo External project
 
+if (NOT PYTHON_EXECUTABLE)
+  message(STATUS "Building libjpeg-turbo requires yasm which requires python to build. Searching for PYTHON_EXECUTABLE")
+  include(CommonFindMacros)
+  fletch_FIND_PYTHON()
```

Comment:
Instead of finding PYTHON here we need to ask the user to explicitly set `yasm_PYTHON_EXECUTABLE`  we also need an else case here that sets `yasm_PYTHON_EXECUTABLE` to `PYTHON_EXCUTABLE` in the case that is is found.